persistence-modules/spring-data-elasticsearch/src/main/java/com/baeldung/elasticsearch/Location.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.elasticsearch;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Just letting you know (for future articles) that the final decision for the use of Lombok has been to avoid it, and rely on the IDE to automatically create the constructors, getters/setters, etc. Since the article already uses Lombok let's keep it.